src/perl/lib/wtsi_clarity/mq/me/flowcell_enhancer.pm

```diff
@@ -285,13 +317,13 @@ sub _build_controls {
 
   # Always will be just the one...
   my %control = ();
-  $control{'sample_uuid'}    = $CONTROL_SAMPLE_UUID;
-  $control{'study_uuid'}     = $CONTROL_STUDY_UUID;
-  $control{'tag_index'}      = $CONTROL_TAG_INDEX;
-  $control{'tag_sequence'}   = $CONTROL_TAG_SEQUENCE;
-  $control{'tag_set_name'}   = $CONTROL_TAG_SET_NAME;
-  $control{'entity_type'}    = $CONTROL_ENTITY_TYPE;
-  $control{'entity_id_lims'} = $self->_spiked_hyb_barcode;
+  $control{'sample_uuid'}     = $CONTROL_SAMPLE_UUID;
+  $control{'study_uuid'}      = $CONTROL_STUDY_UUID;
+  $control{'tag_index'}       = $CONTROL_TAG_INDEX;
+  $control{'tag_sequence'}    = $CONTROL_TAG_SEQUENCE;
+  $control{'tag_set_name'}    = $CONTROL_TAG_SET_NAME;
+  $control{'entity_type'}     = $CONTROL_ENTITY_TYPE;
+  $control{'id_library_lims'} = $self->_spiked_hyb_barcode;
```

Comment:
I can't find `id_library_lims` in the required JSON, but there might be missing fields:
`tag_set_id_lims` and `entity_id_lims`. I might have a wrong version of the required JSON?
I checked this spec: https://github.com/sanger/unified_warehouse/blob/next_release/spec/models/flowcell_spec.rb